<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="COM">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Open_COM">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
     </font>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Flash_Name">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>400</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Open_Flash">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
     </font>
    </property>
    <property name="text">
     <string>载入Flash文件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FLM_TEXT">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>400</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Open_FLM">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
     </font>
    </property>
    <property name="text">
     <string>选择芯片FLM文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Write_Offine">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>写入烧录器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Output_Conf">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>200</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>导出配置文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Input_Conf">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>导入配置文件</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>520</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="File_Name">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Default_MCU</string>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::VisualMoveStyle</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Name_lab">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>镜像注释</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Refresh_COM">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
     </font>
    </property>
    <property name="text">
     <string>刷新串口</string>
    </property>
   </widget>
   <widget class="QLabel" name="Designer">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>360</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Designer : airtech</string>
    </property>
   </widget>
   <widget class="QLabel" name="Hash">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>300</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Flash文件MD5: </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="About"/>
    <addaction name="Help"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="About">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="Help">
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
